src/resolvers/Mutation/createPost.ts

```diff
@@ -93,6 +94,38 @@ export const createPost: MutationResolvers["createPost"] = async (
     imageUrl: args.file ? uploadImageObj?.newImagePath : "",
   });
 
+  // Check if the pinned property is set
+  if (args.data.pinned) {
+    // Check if the user has privileges to pin the post
+    const currentUser = await User.findOne({
+      _id: context.userId,
+    }).lean();
+    if (
+      !(
```

Comment:
This comparision is hard to read. Simplify it to this:-
```
const currentUserIsOrganizationAdmin = currentUser.adminFor.some(organzationId=> organizationId.toString() === args.data.organizationId))

if (currentUser.userType !== "SUPERADMIN" && !currentUserIsOrganizationAdmin){
  throw error.....
}
```

Reply:
Got it.